sympy/stats/tests/test_continuous_rv.py

```diff
@@ -631,9 +631,12 @@ def test_gumbel():
     beta = Symbol("beta", positive=True)
     mu = Symbol("mu")
     x = Symbol("x")
+    y = Symbol("y")
     X = Gumbel("x", beta, mu)
+    Y = Gumbel("y", beta, mu, minimum=True)
     assert str(density(X)(x)) == 'exp(-exp(-(-mu + x)/beta) - (-mu + x)/beta)/beta'
-    assert cdf(X)(x) == exp(-exp((mu - x)/beta))
+    assert str(density(Y)(y)) == 'exp(-exp((-mu + y)/beta) + (-mu + y)/beta)/beta'
+    assert str(cdf(X)(x)) == 'exp(-exp(-(-mu + x)/beta))'
```

Comment:
string comparison is not recommended

Reply:
I have `.expand()` as nothing else was working out.